tests/config/paragraph-spans.json

```diff
@@ -3,10 +3,10 @@
     {
       "name": "paragraph-spans-test",
       "documents": [
-        "s3://ai2-llm/pretraining-data/tests/mixer/inputs/v0/documents/*/0000.json.gz"
+        "tests/data/provided/documents/*"
```

Comment:
I'm a little confused. Are the tests reading writing to S3? Some of them? I see some S3 uploading happening in the Python `TestCase`s.

My preference would be to have most of the tests operating on purely local data, to fully test the logic in the mixer and deduper. There should be at least one test that reads/writes from S3, just to ensure that's working, but it doesn't have to verify any internal logic.

Reply:
all tests run with local file by default; further, `test_dedupe_by_url_remote_input`, ` test_local_input_remote_output`, `test_remote_input_remote_output`, and `test_remote_input_local_output` run on a combination of local/remote input/output.